rDNA/R/rDNA.R

```diff
@@ -7109,6 +7109,7 @@ dna_plotDendro <- function(clust,
 #' @author Philip Leifeld, Johannes B. Gruber
 #'
 #' @examples
+#' \dontrun{
```

Comment:
Why should this not run? Does it take too long? Does the CRAN check complain about that?

Reply:
`dna_dendrogram` does not run with `--as-cran` (see above). The error is `Fatal error: length > 1 in coercion to logical`, but it does not say in which line this happens. Since the function is quite long and contains many `if` statments, I didn't debug it yet.

I tried running the examples for both `dna_cluster` and `dna_plotDendro`. All examples there run without a problem. We can probably run the examples once the test for `dna_dendrogram` is working.

Reply:
In fact, I just removed all `\dontrun{}` but this one and the checks still run ok. Should we run all examples from now on?

Reply:
I removed `\dontrun{}` except in the examples in `dna_dendrogram`. I think it's a different issue why those do not run.

Reply:
Great, thanks! Can I merge, or is there anything else you need to take care of in this PR?

Reply:
I think this is done! Except you have any other issue with it.